chatminer/chatparsers.py

```diff
@@ -237,12 +238,32 @@ def _read_file_into_list(self):
 
     def _parse_message(self, mess):
         parsed_message = {
-            "datetime": mess["from"],
-            "author": datetime.datetime.fromtimestamp(int(mess["date_unixtime"])),
+            "author": mess["from"],
+            "datetime": datetime.datetime.fromtimestamp(int(mess["date_unixtime"])),
             "message": mess["text"],
         }
         return parsed_message
 
+class TelegramHtmlParser(Parser):
+    def _read_file_into_list(self):
+        with self._file.open(encoding="utf-8") as f:
+            soup = BeautifulSoup(f, "html.parser")
```

Comment:
why not use the [native Python HTML parser](https://docs.python.org/3/library/html.parser.html)? BeautifulSoup unnecessarily pollutes the dependencies.

Reply:
The native one doesn't have the find capability using class, tag, etc, and other types of traversals afaik. 
I have used BeautifulSoup before, and also it uses `html.parser` for traversing anyway (which is the native HTML Parser you were talking about).